downloads/swan-lake-release-notes/swan-lake-2201.8.0/RELEASE_NOTE.md

```diff
@@ -140,6 +140,52 @@ If you have not installed Ballerina, download the [installers](/downloads/#swanl
     3d|4f|6 b = 4; // Compilation error now, `4` is considered to be `int`.
     ```
 
+- A bug where an `anydata` value could be assigned to contexts requiring an `anydata & readonly` value has been fixed.
+
+    ```ballerina
+    public function main() {
+        anydata a = [];
+        anydata & readonly b = a; // Compilation error now.
+    }
+    ```
+  
+- A bug that allowed including a `readonly` `service` object inside a `readonly` non-`service` class has been fixed.
+
+    ```ballerina
+    public type A readonly & service object {
+        isolated remote function execute() returns int;
+    };
+  
+    readonly client class B {
+        *A; // Compilation error now.
+  
+        isolated remote function execute() returns int {
+            return 2;
+        }
+    }
+    ```
+
+- A bug where the runtime error messages were including the flattened representation of the union types instead of the name used in the type definition has been fixed.
+  
+    ```ballerina
+    type AC map<int>|string;
+    
+    public function main() returns error? {
+        map<anydata> x = {a: "Ballerina"};
+        AC|boolean _ = check x.cloneWithType();
+    }
+    ```
+
+    This now fails with ```"'map<anydata>' value cannot be converted to '(AC|boolean)'"``` instead of ```"'map<anydata>' value cannot be converted to '(map<int>|string|boolean)'"```.
+
+- Modified the behavior of the [runtime Java APIs to support the intersection type](/downloads/swan-lake-release-notes/swan-lake-2201.8.0#intersection-type-support-in-runtime-java-apis).
+
+- Modified the behavior of the Semantic API `typeDescriptor()` methods to return the intersection type symbol instead of the effective type symbol (i.e., the symbol of the corresponding non-intersection type that is computed to represent the same value space).
```

Comment:
Shouldn't these be in the form of "The behaviour of ... has been modified ..." for consistency? 

@praneesha

Reply:
We can have that format only for bugs as a special case as the meaning gets changed in that instance. This sounds fine to be consistent with the other sections.